Source/Core/VideoCommon/ShaderCache.cpp

```diff
@@ -627,6 +617,22 @@ static GXPipelineUid ApplyDriverBugs(const GXPipelineUid& in)
     ps->ztest = EmulatedZ::Early;
   }
 
+  // We can use framebuffer fetch to emulate logic ops in the fragment shader.
+  // So don't use the worse blend approximation.
+  if (blend.logicopenable && !g_ActiveConfig.backend_info.bSupportsLogicOp &&
+      !g_ActiveConfig.backend_info.bSupportsFramebufferFetch)
+  {
+    if (!blend.LogicOpApproximationIsExact())
```

Comment:
Why isn't the log valid always?  We're still approximating logic ops with blending regardless?

Reply:
Oh I see, `ApproximateLogicOpWithBlending` doesn't mean what it says anymore :(